config/zephyr/chip-module/Kconfig.defaults

```diff
@@ -40,9 +40,11 @@ config LOG_DEFAULT_LEVEL
 
 endif
 
+# disable synchronous printk to avoid blocking IRQs which
```

Comment:
fyi: we don't use this file (it's for standalone zephyr platform in Matter)

Reply:
Yes, I know. I thought i would be more consistent anyway.